.circleci/config.yml

```diff
@@ -2,7 +2,7 @@
 version: 2.1
 
 orbs:
-  architect: giantswarm/architect@0.8.1
+  architect: giantswarm/architect@dev:deploy-without-tag
```

Comment:
Will update to `0.8.3` once https://github.com/giantswarm/architect-orb/pull/103 is merged.